ci/Jenkinsfile.tests-e2e

```
@@ -249,3 +249,11 @@ def getDefaultTestScopeFlag() {
     return '-m=critical'
   }
 }
+
+def getTestStageTimeout() {
+  if (TEST_SCOPE_FLAG == '-m=critical') {
+    return 30
+  } else {
+    return 120
+  }
+}
```

This could be one line:
```suggestion
def getTestStageTimeout() {TEST_SCOPE_FLAG == '-m=critical' ? 30 : 120 }
```